### ISSUE TITLE
Add smoothing "spin-down delay" and/or "spin-down decay rate" options

### ISSUE THREAD

**Issue by chaosblad3** [Shadlock0133/music-vibes#8]
After a detected noise raises the vibration intensity to the appropriate % it should be possible to configure both an initial delay in ms at which the program will sustain the maximum intensity reached before it begins to fall (e.g. a noise raises the vibe to 50% and is then configured to hold it there after the noise ends for a ~100ms delay) which is then followed by a configurable decay-over-time rate that will cause the vibration % to gradually fall at a set rate instead of just immediately dropping to match the current audio level, at least until a new noise overrides it and raises the intensity back up again.

Obviously setting these values to 0 would disable all of this functionality and make it immediately respond to both rises and falls in the audio level as it does currently, but for certain applications (e.g. when using a minimum cut-off value, or where the music just has lots of ups and downs) it could be useful to have the intensity linger and/or decay after each burst of noise to some degree.

These settings could either be configured globally or on a per toy/motor basis.

I hope all of that makes sense ;)

**Comment by Shadlock0133** [comment]
good idea, but i don't have much time/motivation to work on this right now, sorry

**Comment by chaosblad3** [comment]
Thanks for implementing this @Tcan96 ;)
